x-pack/plugins/enterprise_search/server/routes/app_search/curations.ts

```diff
@@ -115,21 +115,4 @@ export function registerCurationsRoutes({
       path: '/as/engines/:engineName/curations/find_or_create',
     })
   );
-
-  router.get(
-    {
-      path: '/api/app_search/engines/{engineName}/curation_search',
```

Comment:
We also need to update the corresponding curation logic file to point at the new `/search` endpoint though right?

Reply:
jk, disregard this comment

Reply:
@JasonStoltz oh wait, you'll probably have to update the tests for this though